server/server.go

```diff
@@ -77,8 +78,13 @@ type serverAuthProxyHandler struct {
 func (h *serverAuthProxyHandler) handler(w http.ResponseWriter, r *http.Request) {
 	statusCode := http.StatusOK
 
-	// before we handle any requests, make sure the user is authenticated
-	if h.credentials.Username != "" || h.credentials.Password != "" {
+	if strings.Contains(r.Header.Get("Authorization"), "Bearer") {
+		err := config.ValidateToken(strings.TrimPrefix(r.Header.Get("Authorization"), "Bearer "))
+		if err != nil {
+			log.Debugf("Error token %+v", err)
```

Comment:
I would move this to Info/Warn.
I guess I would like to see in the log bad authentication entries.

Reply:
Changed to Warning